addOns/automation/src/main/resources/org/zaproxy/addon/automation/resources/Messages.properties

```diff
@@ -438,6 +438,8 @@ automation.out.title.fail = Automation plan failures:
 automation.out.title.good = Automation plan succeeded!
 automation.out.title.warn = Automation plan warnings:
 
+automation.output.unsavedPlan = Unsaved Automation Plan
```

Comment:
If this is going to be a single panel for potentially multiple plans (as you mentioned above), then should this be "Plan(s)"?